README.md

```diff
@@ -157,67 +173,76 @@ TBD
 
-* Version update to 1.2.177-5
+* Version update to 1.2.177-5 ([Column bug](#column-bug))
 
   - Platypus update 0.8.1 to 0.8.1.1 (this only fixes the reported Platypus version number in the VCF)
   - Added conda environment
   - Reordered cvalues in plugin XML configuration
   - Neutral refactoring in `indelCalling.sh`
   - Bugfix: JSON syntax for "file"
 
-* Version update to 1.2.177-4 (includes development versions 1.2.177-1 to -3)
+* Version update to 1.2.177-4 (includes development versions 1.2.177-1 to -3) ([Column bug](#column-bug))
 
   - Roddy 3.0 support
   - Updated COWorkflows base plugin from 1.2.59 to 1.2.76
   - Fixed reported version
   - Bugfix in `indel_extractor_v1.pl` concerning ncRNA_exonic and ncRNA_splicing annotations for somatic calls.
-  - Added `runTindo` cvalue
+  - Added `runTinda` cvalue
 
-* Version update to 1.0.176-10 (=1.2.177-1)
+* Version update to 1.0.177-2 ([Column bug](#column-bug))
+
+  - Bugfix ncRNA file
+  - Allow toggling on/off of Tinda
+
+* Version update to 1.0.176-10 (=1.0.177-1) ([Column bug](#column-bug))
 
   - Added right and bottom border features to `TiN_canopyBasedClustering.R`
   - Version is not backwards compatible
 
-* Version update to 1.0.176-9 (=1.0.177)
+* Version update to 1.0.176-9 (=1.0.177) ([Column bug](#column-bug))
 
   - Fix of a bug affecting versions 1.0.176 to 1.0.176-8 that results in higher false positive rate.
 
-* Version update to 1.0.176
+> NOTE: Refrain from using versions 1.0.176 through 1.0.176-8. They are affected by a bug that increases the false positive rate.
+
+* Version update to 1.0.176 ([Column bug](#column-bug))
 
   - SNVs calling made default.
   - Swapchecker - checks for tumor/control swap from the same PID.
   - TiNDA - Tumor in normal detection analysis, using Canopy's EM-clustering algorithm.
 
-* Version update to 1.0.168
+* Version update to 1.0.168 ([Column bug](#column-bug))
 
   - Further checks for the platypus indel calling step are introduced. A zgrep will be performed, together with a linecount to see if there are any faulty lines in the raw vcf file.
-  
-* Version update to 1.0.167
 
-  - Note that commit e8d8fc27 _introduced_ a bug. Output VCF with swapped control and tumor genotype columns if they are in the 11th and 10th column respectively.
+* Version update to 1.0.167 (<a id="column-bug"></a>Column bug)
+
+  - Note that commit e8d8fc27 _introduced_ a bug that caused the labels of the control and tumor genotype columns (10 & 11) to be swapped into alphabetical order, but left the data in the original order. The root cause of the bug is that Platypus reorders these columns alphabetically, but that this was not accounted for in the workflow code. The bug has no influence on the list of reported indels. This means that all somatic indels are correct. Only sample pairs are affected for which the alphabetical order of the tumor sample is before that of the control sample (e.g. cell_line < control).
```

Comment:
@vinjana: could you please add/include this comment as well:

When the platypus reorders the sample columns alpha-numerically, workflow detects the correct control and tumor sample columns for the somatic calls. But because of the bug _introduced_ in the commit e8d8fc27, it leaves the original (alpha-numeric) sample column orders in the resulting vcf file. This will affect the downstream analysis, which uses column order/number to detect control and tumor genotypes.
To reiterate, all somatic indels are correct, only sample pairs are affected for which the alpha-numeric order of the tumor sample is before that of the control sample (e.g. cell_line < control).